packages/editor/src/components/rich-text/index.js

```diff
@@ -290,7 +301,7 @@ export class RichText extends Component {
 				mode: 'BLOCKS',
 				tagName: this.props.tagName,
 			} );
-			const shouldReplace = this.props.onReplace && this.isEmpty();
+			const shouldReplace = this.props.onReplace && isRichTextValueEmpty( this.props.value );
```

Comment:
@tofumatt Shouldn't this be `this.isEmpty`? OR should we replace them all and remove `this.isEmpty` entirely?

Reply:
Oops, you’re right. I shouldn’t have replaced this one.